### ISSUE TITLE
java.util.logging Does Not Work Out of the Box 

### ISSUE THREAD

**Issue by kittylyst** [oracle/graal#2370]
**Describe the issue**
The document at https://www.graalvm.org/docs/reference-manual/languages/Native-Image/user/LOGGING describes j.u.logging as working out of the box at INFO level on console. It does not appear to.

**Steps to reproduce the issue**
1. git clone https://github.com/kittylyst/jfr-parser_repro_case_graalvm_linkage_bug-bevans.git
2. mvn clean package
3. ./target/JFRFileParser <JFR File>

**Describe GraalVM and your environment:**
- GraalVM version - 20.0 release & latest snapshot
- JDK major version: 11
- OS: macOS Catalina
- Architecture: AMD64

**More details**
Output from mvn -X clean package

[mvn.txt](https://github.com/oracle/graal/files/4505809/mvn.txt)

**Comment by jvican** [comment]
> The document at [graalvm.org/docs/reference-manual/languages/Native-Image/user/LOGGING](https://www.graalvm.org/docs/reference-manual/languages/Native-Image/user/LOGGING) describes j.u.logging as working out of the box at INFO level on console. It does not appear to.

Yeah, I agree. Looks like you might need to add something like this to a logging-related class whose static initializers will be run by GraalVM Native. I personally added this in the implementation of my own slf4j adapter because I own it.

```java
	public static void loadGraalVMNativeConfig() {
		try (InputStream is = ScribeLoggerFullAdapter.class.getClassLoader()
				.getResourceAsStream("graalvm-native-logging.properties")) {
			if (is != null) {
				LogManager.getLogManager().readConfiguration(is);
			}
		} catch (IOException ignore) {
		}
	}

	static {
		if (System.getProperty("java.util.logging.config.file") == null) {
			loadGraalVMNativeConfig();
		}
	}
```

After doing something like this and adding a file `graalvm-native-logging.properties` to your resources directories, the logging should work. Here's an example of such a file to show critical debug information about Netty configuration:

**Comment by jvican** [comment]
I think it would be a good idea for GraalVM Native to support an extra file other than `logging.properties`, maybe called `graalvm-native-logging.properties`, to put all of the logging related information in there. I think that would simplify the setup tremendously. I think the docs should also be updated as right now they are not really useful.
